### DISCUSSION TITLE
Improvement of the display of the basic version of Bambu Lab Studio in OrcaSlicer and in Github.

### DISCUSSION THREAD

**Discussion by fr3ak2402** [OrcaSlicer/OrcaSlicer#961] [Ideas]
@SoftFever 
Personally, I have noticed that many people are a bit confused when they download OrcaSlicer V1.6.2 and then find that the functions of Bambu Studio V1.6.2 are missing.

Because yes at the beginning of the program is based on Bambu Studio and the version numbers of both programs are the same. Seems that for many that OrcaSlicer is based on the Bambu Studio V1.6.2, which it does not.

So I have the following suggestion to make this a little more understandable for users. 

The OrcaSlicer has an info window that looks like this:

<img width="637" alt="Bildschirmfoto 2023-05-05 um 12 58 19" src="https://user-images.githubusercontent.com/96239814/236442545-cdacd3c5-a7c0-45bd-8804-38b5e91089b4.png">

Is it not possible to insert a paragraph at this position, which gives the following information:

Bambu Studio V 1.6.2 Beta2 (v01.06.02.04)

<img width="637" alt="Bildschirmfoto 2023-05-05 um 12 58 19" src="https://user-images.githubusercontent.com/96239814/236443688-8e56adbc-854b-4aa8-87b8-7d1fa9a4ef5c.png">

Or what would be even easier would be to change the sentence Based on Bambu Studio and PrusaSlicer so that the following sentence is there:

Based on Bambu Studio (V 1.6.2 Beta2) and PrusaSlicer 

<img width="637" alt="Bildschirmfoto 2023-05-05 um 12 58 19" src="https://user-images.githubusercontent.com/96239814/236444065-0b86b381-6144-496c-9a85-19d40af66e42.png">

In addition, you could also write in the Github with each new release, at this point, on which version of the OrcaSlicer from Bambu Studio is based.

<img width="874" alt="Bildschirmfoto 2023-05-05 um 13 21 50" src="https://user-images.githubusercontent.com/96239814/236445219-4897729d-aa69-45d9-9dda-2960f952ed46.png">

**Comment by Photogad** [comment]
+ 1
